ChangeLog

```diff
@@ -7,6 +7,11 @@ What's New in Pylint 2.0?
 
 Release date: tba
 
+	  * Adding new check 'mutable-sequence-modified-in-loop' when a mutable
```

Comment:
The indentation seems a bit off, as if it uses tabs. Please use the same format as the rest of the file. Also, let's rephrase this to ``Added a new check, 'mutable-sequence-modified-in-loop``, followed by an empty line, followed by ``This is emitted when a mutable sequence is modified in a for loop. Close #1297``